ruby_event_store/lib/ruby_event_store/global_subscription.rb

```diff
@@ -0,0 +1,59 @@
+module RubyEventStore
+  class GlobalSubscription
+    def initialize(subscriber, store)
+      @subscriber = subscriber
+      @store = store
+      @store.add(self)
+    end
+
+    def call(event)
+      (Class === subscriber ? subscriber.new : subscriber).call(event)
+    end
+
+    def unsubscribe
+      @store.delete(self)
+    end
```

Comment:
with `#call` aliased to `#unsubscribe` this would not be a breaking change, but `#call` is already taken...